auth:
  staff:
    owners:
      logins:
      - robot-maps-sandbox
      groups:
      - svc_vops_maps
      - svc_maps-core-common-balancer
labels:
  balancer_deployer_commit_id: '9754591'
  balancer_deployer_sign: 6318e237fbbd62be70db89791d3f5e58
  order: '40000000'
---
regexp_section:
  matcher:
    match_fsm:
      host: 'core-nmaps-mrc-ugc-back\\.common\\.testing\\.maps\\.yandex\\.uz'
  modules:
    - report:
        uuid: 'core-nmaps-mrc-ugc-back_common_testing_maps_yandex_uz'
        ranges: default
    - report:
        uuid: 'core-nmaps-mrc-ugc-back_common_testing_maps_yandex_uz_default'
        ranges: default
    - threshold:
        lo_bytes: 734003
        hi_bytes: 838860
        pass_timeout: 10s
        recv_timeout: 1s
    - headers:
        append_func:
          X-Forwarded-For: realip
        create_func_weak:
          X-Real-IP: realip
          X-Source-Port-Y: realport
        create:
          X-Ya-Backend-Timeout-Ms: '5000'
    - balancer2:
        attempts: 3
        active:
          steady: true
          delay: '10s'
          request: 'GET /ping HTTP/1.1\nHost: core-nmaps-mrc-ugc-back.common.testing.maps.yandex.uz\r\n\r\n'
        attempts_rate_limiter:
          limit: 0.34
        generated_proxy_backends:
          proxy_options:
            connect_timeout: 70ms
            backend_timeout: 5000ms
            keepalive_count: 100
            fail_on_5xx: true
          include_backends:
            type: BY_ID
            ids: [maps_core_nmaps_mrc_ugc_back_testing_sas]
        on_error:
          errordocument:
            status: 504
            content: "Service unavailable"
